tests/cmd/stability/main.go

```diff
@@ -230,12 +230,18 @@ func main() {
 	// stop one etcd node and k8s/operator/tidbcluster is available
 	faultEtcd := tests.SelectNode(conf.ETCDs)
 	fta.StopETCDOrDie(faultEtcd)
-	defer fta.StartETCDOrDie(faultEtcd)
```

Comment:
@xiaojingchen ptal

Reply:
it's ok, we recover all etcd when test begins